src/projections/gui/MainWindow.java

```diff
@@ -442,44 +442,28 @@ protected void closeAll() {
 		resetStatus();
 	}
 
+	private String generateInfoPanelHTML(String name, String value) {
+		if (value != null)
+			return "<tr><td style=\"text-align: right\">" + name + ":</td><td>" + value + "</td>";
```

Comment:
where is the row closed?

Reply:
I knew I was going to forget something, I just pushed a fix.